tasks.py

```diff
@@ -328,7 +328,9 @@ def export_records(c, filename='data.json'):
     with open(filename, "w") as f_out:
         f_out.write(json.dumps(data, indent=2))
 
-    print("Data export completed")
+    print("Data export completed. Removing temporary files")
+
+    os.remove(tmpfile)
```

Comment:
Suggestion: I would appreciate adding another switch for deleting the tempfile: it contains the permissions - that might be useful for some users.

Reply:
It's something I considered when looking at the code. In my case, looking through the tmp-file did not make any substantial changes concerning permissions - it did however change 'special characters' (ä,ö,...) and indentation of the file.

I don't understand enough about Drupal to judge myself - but shouldn't the permissions part be covered by the excludes defined in `content_excludes()`, instead of being removed afterwards?

Anyway, having two files, one called .tmp, is not really a transparent solution. Having a switch may make sense though.

Reply:
We are using django. Excluding permissions in the export leads to custom user-model implementations breaking outright. That is why it needs to be done afterwards as that only removes the specified classes.
We have a sizeable user base. I prefer not to break the cli for them.

Reply:
Yes, Django is what I meant *facepalm*

What you say makes sense. I have added two flags to allow the behaviour I think makes sense, with flags, and the historical behaviour without the flags.

Reply:
Looks good. Thx!